test/parallel/test-fs-options-immutable.js

```diff
@@ -58,9 +58,7 @@ if (common.canCreateSymLink()) {
   );
 }
 
-if (!common.isAix) {
-  // TODO(thefourtheye) Remove this guard once
-  // https://github.com/nodejs/node/issues/5085 is fixed
+{
```

Comment:
Well IMHO it's excessive since it doesn't (directly) provide scoping for any variables (or any code for that matter). I don't know how anyone else feels about it.